static/app/views/insights/pages/domainViewHeader.tsx

```diff
@@ -123,19 +129,21 @@ function TabLabel({moduleName}: TabLabelProps) {
   const moduleTitles = useModuleTitles();
   const organization = useOrganization();
   const showBusinessIcon = !isModuleEnabled(moduleName, organization);
-  if (showBusinessIcon) {
+
+  if (showBusinessIcon || isModuleConsideredNew(moduleName)) {
     return (
-      <TabWithIconContainer>
+      <TabContainer>
         {moduleTitles[moduleName]}
-        <IconBusiness />
-      </TabWithIconContainer>
+        {isModuleConsideredNew(moduleName) && <Tag>{t('New')}</Tag>}
```

Comment:
I think this'd be better using [`<Badge type="new"`](https://sentry.sentry.io/stories/?name=app%2Fcomponents%2Fbadge%2Fbadge.stories.tsx&query=badge) 👀

Reply:
Thanks for pointing out, looks prettier now too :)